web/config.py

```diff
@@ -513,10 +513,10 @@
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 
 ##########################################################################
-# Number of records to fetch in one batch in query tool when query result
-# set is large.
+# Number of records to fetch in one page in query tool when query result
+# set is large and is divided in multiple pages
 ##########################################################################
-ON_DEMAND_RECORD_COUNT = 1000
+DATA_RESULT_PAGE_SIZE = 1000
```

Comment:
Rename this to **RECORDS_PER_PAGE** or **ROWS_PER_PAGE** instead of DATA_RESULT_PAGE_SIZE. Page size word misleads.

Check and Change in all the files.

Reply:
Page size is a widely used term in pagination.

Reply:
Reanmed to `DATA_RESULT_ROWS_PER_PAGE`